scipy/stats/__init__.py

```diff
@@ -194,6 +194,22 @@
    zipf                     -- Zipf (Zeta)
    zipfian                  -- Zipfian
 
+Censored data class
+-------------------
+
+The univariate continuous distributions can be fit to censored data
+by maximum likelihood estimation.  Censored data is represented with
+instances of the `CensoredData` class.
+
+.. autosummary::
+   :toctree: generated/
+
+   CensoredData
+
+
+Statistical functions
+=====================
```

Comment:
Yes, this needs some work.  An alternative would be to use `-----` in all the headings that follow.  I'll experiment to see what looks good.

Reply:
I made the change that I suggested in my previous comment.  I think it looks OK.

Reply:
I'm not sure about this. Does it make sense for "Statistical functions" (here) to be the name of a a heading within "Statistical Functions" (above). Is `CensoredData` a "Probability Distribution"? Does the QMC submodule contain "Statistical Functions" more than "Probability Distribution"s? I would prefer not to reorganize the whole stats page in this PR. Let's find a reasonable place to put `CensoredData` and think about the rest separately.

Reply:
I moved the description of the CensoredData class to the end of 'Continuous distributions' subsection, since it is only relevant to those distributions.  I also reverted the various changes to the headings and heading markup.